x-pack/plugins/security_solution/public/cases/components/all_cases/index.tsx

```diff
@@ -240,7 +241,10 @@ export const AllCases = React.memo<AllCasesProps>(
             });
           }
         }
-        const convertToDeleteCases: DeleteCase[] = caseIds.map((id) => ({ id }));
+        const convertToDeleteCases: DeleteCase[] = caseIds.map((id) => ({
+          id,
+          type: CaseType.individual,
```

Comment:
Not sure if we can set type to individual here. We are still able to bulk delete case collection, right?

Reply:
Oh yes, you are right. I forgot about collections.